### PR TITLE
chore(rtl): clearly state that rtl and ltr cannot be mixed

### PR DESCRIPTION
**Pull request by ntacheva**

### CODE REVIEW

File: globalization/rtl-support.md [hunk 1]
```diff
@@ -16,7 +16,9 @@ The default language input for most users on the web is left-to-right. However,
 
 ## Configuration
 
-Enabling the right-to-left support is configured on the root level so it targets all UI for Blazor components in the application. To enable the Right-to-Left direction of the components in your application, set the `EnableRtl` parameter of the [`TelerikRootComponent`]({%slug rootcomponent-overview%})  to `true`.
+Enabling the right-to-left support is configured on the root level so it affects all UI for Blazor components in the application. You cannot mix right-to-left and left-to-right orientation in the same application.
```
Comment (dimodi) [inline globalization/rtl-support.md:19]:
```suggestion
Right-to-left support is configured at the root level so it affects all UI for Blazor components in the application. You cannot mix right-to-left and left-to-right orientation in the same application.
```

File: globalization/rtl-support.md [hunk 2]
```diff
@@ -16,7 +16,9 @@ The default language input for most users on the web is left-to-right. However,
...
+
+To enable the Right-to-Left direction of the components in your application, set the `EnableRtl` parameter of the [`TelerikRootComponent`]({%slug rootcomponent-overview%})  to `true`.
```
Comment (dimodi) [inline globalization/rtl-support.md:21]:
```suggestion
To enable right-to-left direction of the components in your application, set the `EnableRtl` parameter of the [`TelerikRootComponent`]({%slug rootcomponent-overview%})  to `true`.
```

**Review by dimodi — APPROVED** [review]

**Review by ikoevska — APPROVED** [review]
Agreed with @dimodi's suggestions.

**Comment by radkostanev** [comment]
Have we had such a request? In theory, there might be valid cases where one could intentionally mix these.

**Pull request merged**